packages/wallet/src/SingleAddressWallet.ts

```diff
@@ -288,6 +295,24 @@ export class SingleAddressWallet implements ObservableWallet {
       tip$: this.tip$,
       transactionsHistoryStore: stores.transactions
     });
+
+    this.#rollbackSubscription.add(
+      createTransactionReemitter({
+        confirmed$: this.transactions.outgoing.confirmed$,
+        rollback$: this.transactions.rollback$,
+        store: stores.volatileTransactions,
+        submitting$: this.transactions.outgoing.submitting$
+      })
+        .pipe(
+          mergeMap((rollbackTransaction) => from(this.submitTx(rollbackTransaction))),
+          catchError((err) => {
+            this.#logger.debug('Failed to resubmit rolled back transaction', err);
```

Comment:
```suggestion
            this.#logger.error('Failed to resubmit rolled back transaction', err);
```

Reply:
This was per @rhyslbw suggestion to use the `.debug` level. I assume any errors would be caught/logged by the normal tx sumit flow. This was just an extra logging step to traceback the error to rollbacks.

Reply:
Depends on which tx submit provider, I think this should be logging an error as this is where the `submitTx` is called :thinking: Thoughts @rhyslbw ?

Reply:
Yeah, we need to consider the appropriate place to log errors. I'm not sure I have a good answer here, as I think it makes sense for the `Provider` to do it, but there's no way to ensure it :thinking: 

Currently, we're not in these, so best to use `error` over `debug` here:
- https://github.com/input-output-hk/cardano-js-sdk/blob/master/packages/blockfrost/src/blockfrostTxSubmitProvider.ts
- https://github.com/input-output-hk/cardano-js-sdk/blob/master/packages/cardano-services-client/src/TxSubmitProvider/txSubmitHttpProvider.ts
- https://github.com/input-output-hk/cardano-js-sdk/blob/980c2227d39af5b2b3476597a4fb74615f4e62aa/packages/ogmios/src/Provider/TxSubmitProvider/ogmiosTxSubmitProvider.ts#L57-L59

Reply:
Done in: [ddb6af8](https://github.com/input-output-hk/cardano-js-sdk/pull/351/commits/ddb6af800a591149f87a2668b4da04d0470a29c7)